### ISSUE TITLE
libpng warning

### ISSUE THREAD

**Issue by kenn** [thoughtbot/paperclip#1552]
I start to see a warning on `identify` and `convert` for PNG as follows:

```
identify: iCCP: Not recognizing known sRGB profile that has been edited `/var/folders/fm/3wqgnvhn42g6kx5m70c2p28m0000gn/T/7b1ec0e50435690ad7072859321398f420140515-10846-1dhsudq' @ warning/png.c/MagickPNGWarningHandler/1832.
convert: iCCP: Not recognizing known sRGB profile that has been edited `/var/folders/fm/3wqgnvhn42g6kx5m70c2p28m0000gn/T/7b1ec0e50435690ad7072859321398f420140515-10846-1dhsudq' @ warning/png.c/MagickPNGWarningHandler/1832.
```

According to the following article, it was due to a newer version of libpng and is safe to ignore.

http://tex.stackexchange.com/a/125746/51814

It seems that we could silence that by adding `-strip` option.

```
convert $i -strip $i
```

Do we want to fix this?

**Comment by maclover7** [comment]
Hi @kenn ! Is this still an issue for you in Paperclip; I know this issue is from approximately 1 year ago. If it is still an issue, can you please provide the code that's causing you the error? Thanks!

**Comment by kenn** [comment]
I don't have an environment to reproduce this issue right now, but it seems that the issue [has been fixed as of libpng 1.6.13](https://code.google.com/p/android/issues/detail?can=2&start=0&num=100&q=&colspec=ID%20Type%20Status%20Owner%20Summary%20Stars&groupby=&sort=&id=77704). Thanks for the heads up!

**Comment by maclover7** [comment]
@jferris @jyurek Please close issue, problem appears to be solved.
